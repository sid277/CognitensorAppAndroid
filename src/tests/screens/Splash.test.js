import React from 'react';
import { shallow } from 'enzyme';
import Splash from '../../screens/Splash';
import renderer from 'react-test-renderer';


test('should test Splash component', () => {
const wrapper = shallow(<Splash />);
expect(wrapper).toMatchSnapshot();
});