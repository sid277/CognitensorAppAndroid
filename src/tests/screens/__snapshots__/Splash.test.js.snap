// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Splash component 1`] = `ShallowWrapper {}`;
